java/core/src/test/java/com/microsoft/bond/testmodel/MyEnum.java

```diff
@@ -78,6 +78,30 @@ public static MyEnum get(int value) {
         }
     }
 
+    public static MyEnum parse(String str) {
```

Comment:
I don't know the requirements. Parse makes sense, but I am not sure about tryParse (which is .NET's naming convention). Java native enums only have method that throws exception (Enum.valueOf). 

I think I should rename "parse" to "valueOf" to keep it close to Java's true enums (and also since we're not really parsing), but I prefer to not add a "try" method at this time.

---
In reply to: [132336715](https://github.com/Microsoft/bond/pull/552#discussion_r132336715) [](ancestors = 132336715)